package.json

```diff
@@ -51,6 +51,7 @@
     "test:integration": "cross-env FORCE_COLOR=1 jest \".*__tests__.*\\.integration\\.(spec|test)\\.ts\" --coverage false",
     "test:system": "cross-env FORCE_COLOR=1 jest \".*__tests__.*\\.system\\.(spec|test)\\.ts\" --coverage false",
     "test:unit": "cross-env FORCE_COLOR=1 jest \".*__tests__.*\\.unit\\.(spec|test)\\.ts\" --coverage",
+    "test:uss": "cross-env FORCE_COLOR=1 jest \".*zosuss.*\\.*__tests__.*\\.(spec|test)\\.ts\" --coverage",
```

Comment:
I would just include your uss tests in the test:system task. We haven't separated any other command groups into their own npm tasks